docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -32,9 +32,44 @@ The PHP API has been enhanced with the following new classes:
 - TODO
 
 [[= release_note_entry_end() =]]
-[[= release_note_entry_begin("Ibexa DXP " + version, 'TODO', ['Headless', 'Experience', 'Commerce']) =]]
+[[= release_note_entry_begin("Ibexa DXP " + version, '2025-08-05', ['Headless', 'Experience', 'Commerce', 'New feature']) =]]
 
-#### TODO
+#### Special characters in online editor
+
+The [online editor](online_editor_guide.md) now allows to easily enter special characters like currency symbols.
+It uses the [special characters plugin](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html),
+
+![Special characters in online editor](4.6_special_characters.png "Special characters in online editor")
+
+#### Pagination for ezobjectrelationlist in GraphQL
+
+To improve performance and gain greater control over the returned responses from the [GraphQL API](graphql.md), you can now [enable pagination](relationlistfield.md#enable-pagination-in-graphql) of relations specified using the Relationlist field type.
+
+#### Taxonomy Subtree limitation
+
+You can now manage access to [taxonomy items](taxonomy.md) more effectively by using the new [Taxonomy Subtree limitation](limitation_reference.md#taxonomy-subtree-limitation).
+
+#### Twig Components
+
+With the latest changes to [Twig Components](components.md), you can:
+
+- set component priority when using YAML configuration
+- render a menu with help of the new Menu component
+
+The list of built-in Twig Component groups has been expanded and includes:
+
+- one new group for the [back office](custom_components.md) (`admin-ui-versions-table-before`)
+- four new groups for [storefront](custommize_storefront_layout.md#customize-with-twig-components)
+
+#### Improved content creation interface
+
+The editing interface of the back office has been improved to better highlight the language, creator, and the publication date when working with content items.
+
+![Improved interface for content creation](4.6_improved_editing.png "Improved interface for content creation")
+
+#### Breaking changes
+
+- The `Ibexa\FieldTypeRichText\RichText\Validator\CustomTagsValidator` class has been renamed to `Ibexa\FieldTypeRichText\RichText\Validator\CustomTemplateValidator`, expanding its responsibility to validate both [custom tags](extend_online_editor.md# configure-custom-tags) and [custom styles](extend_online_editor.md#configure-custom-styles)
```

Comment:
Looks ok, but should there be a space in the link, between anchor and anchor-id?
```suggestion
- The `Ibexa\FieldTypeRichText\RichText\Validator\CustomTagsValidator` class has been renamed to `Ibexa\FieldTypeRichText\RichText\Validator\CustomTemplateValidator`, expanding its responsibility to validate both [custom tags](extend_online_editor.md#configure-custom-tags) and [custom styles](extend_online_editor.md#configure-custom-styles)
```

Reply:
<img width="220" height="110" alt="obraz" src="https://github.com/user-attachments/assets/7af5c0d0-389d-43c3-8851-e9d907e239c0" />

Fixed in https://github.com/ibexa/documentation-developer/pull/2869/commits/97ea04cc9df478449a5ceb5cf7d4259aa842251e , thanks!